<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>677</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbChooseTask">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>311</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>Доступные задачи:</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="gbFibonacci">
         <property name="title">
          <string>Вычисление N-ого числа Фибоначчи</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>221</width>
            <height>51</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QRadioButton" name="rbFibonacci">
             <property name="text">
              <string>число Фиббоначчи</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbFibonacci">
             <property name="maximum">
              <number>60000</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbFactorial">
         <property name="title">
          <string>Вычисление факториала числа N</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>221</width>
            <height>51</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QRadioButton" name="rbFactorial">
             <property name="text">
              <string>Факториал числа</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbFactorial">
             <property name="maximum">
              <number>60000</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbDoubleFactorial">
         <property name="title">
          <string>Вычисление двойного факториала числа N</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>261</width>
            <height>51</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QRadioButton" name="rbDoubleFactorial">
             <property name="text">
              <string>Двойной факториал числа</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbDoubleFactorial">
             <property name="maximum">
              <number>60000</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pbAddTask">
       <property name="text">
        <string>Добавить задачу на выполнение</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="gbTaskQueue">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>691</width>
      <height>621</height>
     </rect>
    </property>
    <property name="title">
     <string>Очередь задач</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="twTasksQueue"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pbStartTaskQueue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>6</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Начать выполнение</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Задайте количество потоков: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbCountThreads">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pbStopTaskQueue">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Завершить выполнение</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
